.go-version

```diff
@@ -1 +1 @@
-1.22.7
+1.22.10
```

Comment:
Hey @sgagniere @channingdong ! Fyi that we're bumping the golang version here due to the dependency requirement:
```
go: github.com/confluentinc/ccloud-sdk-go-v2/iam-ip-filtering@v0.4.0: module github.com/confluentinc/ccloud-sdk-go-v2/iam-ip-filtering@v0.4.0 requires go >= 1.22.10 (running go 1.22.7)
```